core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -94,9 +113,7 @@ public void getStatus(GetStatusPRequest request,
                         StreamObserver<GetStatusPResponse> responseObserver) {
     try {
       String ufsFilePath = request.getPath();
-      UnderFileSystem ufs = UnderFileSystem.Factory.create(request.getPath(),
-          UnderFileSystemConfiguration.defaults(Configuration.global()));
-      UfsFileStatus status = ufs.getFileStatus(request.getPath());
+      UfsFileStatus status = mUfsFileStatusCache.get(request.getPath());
       if (status == null) {
```

Comment:
in which situation this would be null? IDE is warning me this always be false

Reply:
Good call, I've check the get status from ufs impl, looks like it won't return a null, so I just remove the check